packages/transaction-controller/src/utils/gas-fees.ts

```diff
@@ -25,8 +29,10 @@ import { SWAP_TRANSACTION_TYPES } from './swaps';
 export type UpdateGasFeesRequest = {
   eip1559: boolean;
   ethQuery: EthQuery;
-  getSavedGasFees: () => SavedGasFees | undefined;
-  getGasFeeEstimates: () => Promise<GasFeeState>;
+  getSavedGasFees: (chainId: Hex) => SavedGasFees | undefined;
```

Comment:
what?..

Reply:
ahh... this is because the dummy placeholder function accepts a chainId, so the types are mismatched and this updates it